tests/integ/modin/frame/test_stack.py

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import modin.pandas as pd
+import numpy as np
+import pytest
+
+from tests.integ.modin.sql_counter import sql_count_checker
+from tests.integ.modin.utils import create_test_dfs, eval_snowpark_pandas_result
+
+
+@pytest.mark.parametrize(
+    "data, index, columns",
+    [
+        ([[0, 1], [2, 3]], ["cat", "dog"], ["weight", "height"]),
+        ([[0, np.nan], [np.nan, 3]], ["cat", "dog"], ["weight", "height"]),
+    ],
+)
+@pytest.mark.parametrize("dropna", [True, False])
+@pytest.mark.parametrize("sort", [True, False])
```

Comment:
Shouldn't we also test stack with level = [0, 1], since column is a MultiIndex?

Reply:
`level != -1` (i.e. non-default) is not implemented, so there is a `test_stack_multiindex_unsupported` and `test_stack_level_unsupported`.

Reply:
Makes sense (see above: got confused since I thought list-like level was supported thanks to frontend code).